chapters/what_is_vulkan.md

```diff
@@ -15,6 +15,7 @@ For those who might not work with C code, there are various [language](https://g
 ## Vulkan and OpenGL
 
 Some developers might be aware of the other Khronos Group standard [OpenGL](https://www.khronos.org/opengl/) which is also a 3D Graphics API. Vulkan is not a direct replacement for OpenGL, but rather an explicit API that allows for more explicit control of the GPU.
+Khronos' [Vulkan Samples](https://github.com/KhronosGroup/Vulkan-Samples) article on ["How does Vulkan compare to OpenGL ES? What should you expect when targeting Vulkan?](https://github.com/KhronosGroup/Vulkan-Samples/blob/master/samples/vulkan_basics.md) offers a more detailed comparison between the two APIs.
```

Comment:
nit: add a space above to separate otherwise markdown can get rendered strangely sometimes

Reply:
Done